test/framework/bootstrap/kind_util.go

```diff
@@ -132,11 +144,11 @@ func loadImage(ctx context.Context, cluster, image string) error {
 
 // copied from kind https://github.com/kubernetes-sigs/kind/blob/v0.7.0/pkg/cmd/kind/load/docker-image/docker-image.go#L168
 // save saves image to dest, as in `docker save`
-func save(ctx context.Context, image, dest string) error {
-	_, _, err := exec.NewCommand(
+func save(ctx context.Context, image, dest string) ([]byte, []byte, error) {
```

Comment:
Might be we can handle the error here, so we could avoid exposing stdout and stderr detail to the callers